### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by zuzu1383** [bionanoimaging/UC2-GIT#108] [Q&A]
Perhaps I am dense in the head. I can't seem to find a document that describes/explains the UC2. 
For example- does it auto focus? How is the specimen(slide) positioned. Sorry if this is the wrong place to put this but I am frustrated. It is an amazing thing but want to understand the nitty-gritty 1st before building.

Also if anyone has done this with the Jetson nano- let me know ( I did see the GUI for the nano- wonderful)

**Comment by beniroquai** [comment]
Hey @zuzu1383 thanks for reaching out! 
Indeed, we have a number of papers. PErhaps the seminal is this https://www.nature.com/articles/s41467-020-19447-9
There are also several talks on youtube describing the system e.g. https://www.youtube.com/watch?v=ivhmaaL3dRM&ab_channel=LIBREhub 

Putting UC2 in one sentence: It's a sekelton for rapid prototying in optics and gives you the necessary buidling blocks to realize an arbitrarily complicated optical system with 3D printed modules. 

I hope this helps :-)

**Comment by zuzu1383** [comment]
The nature article was exactly what I was looking for. Concise and clear.
The concept and it's evolution is quite remarkable. Can't wait to build
(need to learn python also). Hopefully I can integrate my Jetson nano.

Thanks,
Ben Lightfoot
